lib/capify-ec2.rb

```diff
@@ -372,13 +425,55 @@ def deregister_instance_from_elb_by_dns(server_dns)
   end
 
   def reregister_instance_with_target_group_by_dns(server_dns, target_group, timeout)
-    # Stubbed method
-    # Return `reregistered`
+    instance = get_instance_by_dns(server_dns)
+
+    # Check if the instance is a VPC instance, if not return
+    if not is_vpc_instance?(instance)
+        puts "[Capify-EC2] Skipping non-VPC instance '#{server_dns}' ('#{instance}') from target group '#{target_group}'..."
+        return []
+    end
+
+    # Sleep, but why?
```

Comment:
Good question. Try removing it?

Reply:
Removed. I've left the other sleeps in that are contained within health check loops. They should be exponential backoffs rather than sleeps but they'll suffice for now.